dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/InjectorBG.mo

```diff
@@ -53,5 +59,7 @@ model InjectorBG "Injector controlled by a the susceptance B and the conductance
     PInjPu = -ComplexMath.real(terminal.V * ComplexMath.conj(terminal.i));
     QInjPu = -ComplexMath.imag(terminal.V * ComplexMath.conj(terminal.i));
 
-annotation(preferredView = "text");
+annotation(preferredView = "diagram",
```

Comment:
Thanks for the annotations, the representation of both injectors looks very nice when included in a model such as in SVarCStandard! But the graphical view you added doesn't corresponds to the view "diagram" so I would prefer to keep the preferredView as text (there is currently nothing in the diagram view...)

Reply:
Agreed